stdlib/3.3/ipaddress.pyi

```diff
@@ -1,201 +1,150 @@
-    def __lt__(self, other): ...
-    def __add__(self, other): ...
-    def __sub__(self, other): ...
-    def __hash__(self): ...
-
-class _BaseNetwork(_IPAddressBase):
-    def __init__(self, address) -> None: ...
-    def hosts(self): ...
-    def __iter__(self): ...
-    def __getitem__(self, n): ...
-    def __lt__(self, other): ...
-    def __eq__(self, other): ...
-    def __hash__(self): ...
-    def __contains__(self, other): ...
-    def overlaps(self, other): ...
-    @property
-    def broadcast_address(self): ...
-    @property
-    def hostmask(self): ...
-    @property
-    def with_prefixlen(self): ...
-    @property
-    def with_netmask(self): ...
-    @property
-    def with_hostmask(self): ...
-    @property
-    def num_addresses(self): ...
-    @property
-    def prefixlen(self): ...
-    def address_exclude(self, other): ...
-    def compare_networks(self, other): ...
-    def subnets(self, prefixlen_diff=..., new_prefix=...): ...
-    def supernet(self, prefixlen_diff=..., new_prefix=...): ...
-    @property
-    def is_multicast(self): ...
-    @property
-    def is_reserved(self): ...
-    @property
-    def is_link_local(self): ...
-    @property
-    def is_private(self): ...
-    @property
-    def is_global(self): ...
-    @property
-    def is_unspecified(self): ...
-    @property
-    def is_loopback(self): ...
-
-class _BaseV4:
-    def __init__(self, address) -> None: ...
-    @property
-    def max_prefixlen(self): ...
+    def __sub__(self: _T, other: int) -> _T: ...
+    if sys.version_info >= (3, 4):
+        @property
+        def is_global(self) -> bool: ...
     @property
-    def version(self): ...
-
-class IPv4Address(_BaseV4, _BaseAddress):
-    def __init__(self, address: object) -> None: ...
+    def is_link_local(self) -> bool: ...
     @property
-    def packed(self) -> bytes: ...
+    def is_loopback(self) -> bool: ...
     @property
-    def is_reserved(self) -> bool: ...
+    def is_multicast(self) -> bool: ...
     @property
     def is_private(self) -> bool: ...
     @property
-    def is_multicast(self) -> bool: ...
+    def is_reserved(self) -> bool: ...
     @property
     def is_unspecified(self) -> bool: ...
     @property
-    def is_loopback(self) -> bool: ...
+    def max_prefixlen(self) -> int: ...
     @property
-    def is_link_local(self) -> bool: ...
+    def packed(self) -> bytes: ...
 
-class IPv4Interface(IPv4Address):
-    network = ...  # type: IPv4Network
-    netmask = ...  # type: IPv4Address
-    hostmask = ...  # type: IPv4Address
-    def __init__(self, address: object) -> None: ...
-    def __eq__(self, other: Any) -> bool: ...
-    def __lt__(self, other: Any) -> bool: ...
-    def __hash__(self) -> int: ...
-    @property
-    def ip(self) -> IPv4Address: ...
+class _BaseNetwork(_IPAddressBase, Container, Iterable[_A], Generic[_A]):
+    network_address: _A
+    netmask: _A
+    def __init__(self, address: object, strict: bool=...) -> None: ...
```

Comment:
Spaces around `=`

Reply:
Sorry about this one. I thought *flake8* would catch this?

Reply:
I think we made it so it doesn't complain about missing spaces, but we haven't taught it to require spaces around `=`  in stubs.